CODEOWNERS

```diff
@@ -1 +1 @@
-*   @nestor-maintainers
+*       @nestor-maintainer
```

Comment:
Isn't the name [nestor-maintainers](https://github.com/orgs/ChauffeurPrive/teams/nestor-maintainers) ?

Reply:
Try also @ChauffeurPrive/nestor-maintainers (that is what Github suggests me)